web/src/app/wizard/WizardRouter.js

```diff
@@ -45,8 +45,9 @@ const styles = {
 
 @withWidth()
 @withStyles(styles)
```

Comment:
withStyles has also been replaced by makeStyles from material-core. You can find an example in most hooks files.

https://github.com/target/goalert/blob/master/web/src/app/services/HeartbeatMonitorList.js#L38